src/prototype_creep_resources.js

```diff
@@ -360,7 +360,9 @@ const filterTransferrables = function(creep, object) {
     return false;
   }
 
-  if (object.structureType !== STRUCTURE_STORAGE && object.energy === object.energyCapacity) {
+  if (object.structureType === STRUCTURE_STORAGE ?
```

Comment:
may also add STRUCTURE_TERMINAL && STRUCTURE_CONTAINER

Reply:
Carries and harvesters will only transfer to structures in the base, so it's unnecessary to check `STRUCTURE_CONTAINER` now.  We have `terminalAvailable` for `STRUCTURE_TERMINAL`.